packages/data-stores/src/reader/queries/use-subscription-manager-site-subscriptions-query.ts

```diff
@@ -0,0 +1,71 @@
+import { useQuery } from 'react-query';
+import { callApi } from '../helpers';
+import { useIsLoggedIn, useIsQueryEnabled } from '../hooks';
+import type { SiteSubscription } from '../types';
+
+type SubscriptionManagerSiteSubscriptions = {
+	subscriptions: SiteSubscription[];
+	page: number;
+	total_subscriptions: number;
+};
+
+type SubscriptionManagerSiteSubscriptionsQueryProps = {
+	filter?: ( item?: SiteSubscription ) => boolean;
+	sort?: ( a?: SiteSubscription, b?: SiteSubscription ) => number;
+	limit?: number;
+};
+
+const callFollowingEndPoint = async (
```

Comment:
I ❤️ the readability and cleanliness of this function.

Reply:
Thank you very much  🙏🙏🙏